src/commands/deploy/genezio.ts

```diff
@@ -327,6 +324,8 @@ export async function genezioDeploy(options: GenezioDeployOptions) {
     for (const frontendUrl of frontendUrls) {
         log.info(colors.cyan(`Frontend URL: ${frontendUrl}`));
     }
+
+    return deployClassesResult;
```

Comment:
Why do you return this? I think is not necessary for this change to work, also it does not make sense as this function does multiple things, not just deploying the classes.

Reply:
Yes. It's a left over. Thanks